CMakeLists.txt

```diff
@@ -2031,7 +2031,7 @@ endif()
 #
 
 configure_file(collectors/python.d.plugin/python.d.plugin.in collectors/python.d.plugin/python.d.plugin @ONLY)
-install(FILES ${CMAKE_BINARY_DIR}/collectors/python.d.plugin/python.d.plugin
+install(PROGRAMS ${CMAKE_BINARY_DIR}/collectors/python.d.plugin/python.d.plugin
```

Comment:
@Ferroin I see ([docs](https://cmake.org/cmake/help/latest/command/install.html#programs))

> Files installed by this form are by default given permissions OWNER_WRITE, OWNER_READ, GROUP_READ, and WORLD_READ if no PERMISSIONS argument is given.

> The PROGRAMS form is identical to the FILES form except that the default permissions for the installed file also include OWNER_EXECUTE, GROUP_EXECUTE, and WORLD_EXECUTE. 

So it makes it 755. Should it be 750 for plugins?

Reply:
We currently set 755 for most plugins unless they need SUID or capabilities, which is probably fine (if they don’t need caps/SUID, the information they can expose is stuff that any user can get at anyway).

Reply:
There’s also the issue that we can’t set ownership cleanly from CMake (we couldn’t with autoconf either), so the permissions need to be updated anyway in a number of cases.